lib/view_model/auth_view_model.dart

```diff
@@ -105,16 +106,14 @@ abstract class AuthViewModelBase with Store {
 
   @action
   Future<void> biometricAuth() async {
-    try {
-      final canBiometricAuth = await _biometricAuth.canCheckBiometrics();
-
-      if (canBiometricAuth) {
-        final isAuthenticated = await _biometricAuth.isAuthenticated();
+    final _flutterLocalAuthenticationPlugin = FlutterLocalAuthentication();
 
-        if (isAuthenticated) {
-          state = ExecutedSuccessfullyState();
-        }
+    try {
+      final authenticated = await _flutterLocalAuthenticationPlugin.authenticate();
```

Comment:
you can keep using `_biometricAuth` instance, since you already modified the code there to use the new package, no need to import it here as well and use it separately